plutarch-extra/Plutarch/Extra/Monad.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+-- | TermCont-related adapters for Plutarch functions.
+module Plutarch.Extra.Monad (
```

Comment:
Sorry, I forgot to comment this previously - but I think this module should be named `Plutarch.Extra.TermCont`.

We should definitely make note so as to add more termcont utils here later: `pletFieldsC`, `ptraceC` and some more come to mind.

Reply:
btw what's up with the `UndecidableInstances` pragma?

Reply:
From previous code I guess. Removed.